src/components/SidebarContent.vue

```diff
@@ -304,26 +317,20 @@ export default {
     },
     callSciCrunch: function(apiLocation, params = {}) {
       return new Promise((resolve, reject) => {
-        // the following controller will abort current search
-        // if a new one has been started
-        if (this._controller) this._controller.abort();
```

Comment:
Was the abort controller not working?

Reply:
It was working but its no longer needed as the response size is a lot smaller now - one dataset vs one page of datasets.

Reply:
oh I see, since the call is per card it is not as important

Reply:
No, now I have decided to support parallel requests, the abort controller will be added back in.